src/Command/SymlinksCommand.php

```diff
@@ -175,41 +178,68 @@ private function symlink($source, $target)
      */
     private function findIn($path)
     {
-        return Finder::create()->ignoreDotFiles(false)->filter($this->getFilterClosure())->followLinks()->in($path);
+        return Finder::create()
+            ->ignoreDotFiles(false)
+            ->sort($this->getSortByPathDepthClosure())
+            ->followLinks()
+            ->in($path);
     }
 
     /**
-     * Returns a closure to filter recursive paths.
+     * Filter nested paths because if a parent is symlinked, all nested paths
+     * will be symlinked automatically as well.
+     *
+     * @param Finder $finder
+     *
+     * @return array
+     */
+    private function filterNestedPaths(Finder $finder)
+    {
+        $parents = [];
+        $result = iterator_to_array($finder);
+
+        /** @var SplFileInfo $file */
+        foreach ($result as $k => $file) {
+            $chunks = explode('/', $file->getRelativePath());
+            array_pop($chunks);
+
+            $parent = implode('/', $chunks);
+
+            if (in_array($parent, $parents)) {
+                $this->output->writeln(
+                    sprintf(
+                        'Skipped <error>%s</error> because parent <error>%s</error> will be symlinked already.',
+                        $file->getRelativePath(),
+                        $parent
+                    )
+                );
+
+                unset($result[$k]);
+            }
+
+            $parents[] = $file->getRelativePath();
+            $parents   = array_unique($parents);
```

Comment:
No, but makes `in_array()` calls faster if the amount of files to check increases :)